SampleComponentLibraryApp/SampleComponentLibraryApp/Buttons/ButtonsViewController.swift

```diff
@@ -52,6 +53,13 @@ class ButtonsViewController: UIViewController, Themeable {
                                                              a11yIdentifier: "a11yPrimary")
         primaryButton.configure(viewModel: primaryViewModel)
 
+        let primaryViewModelDisabled = PrimaryRoundedButtonViewModel(
+            title: "Primary Disabled",
```

Comment:
```suggestion
            title: "Primary Disabled",
```
do we need to make these strings go inside strings files instead of directly adding the string here?

Reply:
adding here as this is part of the sample component library